infrastructure/validators/offenses-slashes.md

```diff
@@ -0,0 +1,232 @@
+  choose who your active validator will be (except in the trivial case of nominating a single
+  validator). Staking allocations are controlled by the [Phragmén algorithm](https://wiki.polkadot.network/docs/learn-phragmen#understanding-phragm%C3%A9n){target=\_blank}.
+
+
+Once a validator gets slashed, it goes into the state as an "unapplied slash". You can check this
+via
+[Polkadot-JS UI](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.polkadot.io#/staking/slashes){target=\_blank}.
+The UI shows it per validator, followed by all the affected nominators and the amounts. While
+unapplied, a governance proposal can be made to reverse it during a 27-day grace period, after which
+the slashes are applied.
+
+A slash may occur under the circumstances below:
+
+1.  Equivocations – A slash of 0.01% is applied with as little as a single evocation. The slashed amount increases to 100% incrementally as more validators also equivocate
+2.  Disputes – This may result from a validator trying to represent the contents of a block falsely. Slashing penalties of 100% may apply
+   
+#### Slash for Equivocation
+
+The following levels of offense are
+[defined](https://research.web3.foundation/Polkadot/security/slashing/amounts){target=\_blank}. However, these
+particular levels are not implemented or referred to in the code or the system; they are meant as
+guidelines for different levels of severity for offenses.
+
+- Level 1: Isolated equivocation slashes a minimal amount of the stake.
+- Level 2: Misconducts unlikely to be accidental but do not harm the network's security to any large
+  extent. Examples include concurrent equivocation or isolated cases of unjustified voting in
+  [GRANDPA](https://wiki.polkadot.network/docs/learn-consensus#finality-gadget-grandpa){target=\_blank}. Slashes a moderately small amount of the stake.
+- Level 3: misconduct that poses severe security or monetary risk to the system or mass collusion.
+  Slashes all or most of the stake behind the validator.
+
+The following are scenarios that build towards slashes under equivocation:
+
+1.  Cloning a server, i.e., copying all contents when migrating to new hardware. This action should
+    be avoided. If an image is desired, it should be taken before keys are generated.
+2.  High Availability (HA) Systems – Equivocation can occur if there are any concurrent operations,
+    either when a failed server restarts or if a false positive event results in both servers being
+    online simultaneously. HA systems are to be treated with extreme caution and are not advised.
+3.  The keystore folder is copied when attempting to copy a database from one instance to another.  
+    It is important to note that equivocation slashes occur with a single incident. This can happen
+    if duplicated keystores are used for only a few seconds. A slash can result in losing nominators
+    and funds, removal from the Thousand Validator Programme, and reputational damage.
```

Comment:
```suggestion
1.  Cloning a server, i.e., copying all contents when migrating to new hardware. This action should
    be avoided. If an image is desired, it should be taken before keys are generated
2.  High Availability (HA) Systems – equivocation can occur if there are any concurrent operations,
    either when a failed server restarts or if a false positive event results in both servers being
    online simultaneously. HA systems are to be treated with extreme caution and are not advised
3.  The keystore folder is copied when attempting to copy a database from one instance to another.  
    It is important to note that equivocation slashes occur with a single incident. This can happen
    if duplicated keystores are used for only a few seconds. A slash can result in losing nominators
    and funds, removal from the Thousand Validator Programme, and reputational damage
```